tips/views.py

```diff
@@ -20,6 +21,16 @@ def add_tip(request):
     return render(request, ADD_TIP_PAGE_ROUTE, {'form': form})
 
 
+def categories_filter(request, category):
+    category_filtered_tips = Tip.objects.filter(category=category)
+    return render(request, CATEGORIES_PAGE_ROUTE, {'cats': category, 'category_filtered_tips': category_filtered_tips})
```

Comment:
I looked at the html file rendered from this def and you don't use the "cats" from the context. do you really need it in the html template?  if so maybe change it to "category"

Reply:
agree with the Lior

Reply:
++

Reply:
Is it fixed?